content/scientist/j-ernest-wilkins-jr.md

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Can this file be moved to replace the current Ernest Wilkins Jr file? Also, we already have a photo uploaded under `/img/uploads/J._Ernest_Wilkins,_Jr._9.jpg`